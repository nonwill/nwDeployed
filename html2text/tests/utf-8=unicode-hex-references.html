<!DOCTYPE html>
<html>
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
		<link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet" type="text/css"/>
		<link href="http://www.essent.nl/favicon.ico" rel="icon" type="image/ico"/>
		<title>Essent</title>
		<style>
		A:link {color:#0098b3;text-decoration: none;}
		A:visited {color:#0098b3;text-decoration: none;}
		A:active {color:#0098b3;text-decoration: none;}
		A:hover {color:#0098b3;text-decoration: none;}
		.lato[class~=lato] {font-family:'Lato', Arial, sans-serif !important;font-weight: 400 !important;}
		/* Client-specific Styles / Resets */
		#outlook a {padding:0;} 
		/* Force Outlook to provide a "view in browser" menu link. */
		body{background-color:white;width:100% !important; margin:0 !important;; padding:0 !important;-webkit-text-size-adjust:none !important; -ms-text-size-adjust:none !important;}
		/* Prevent Webkit and Windows Mobile platforms from changing default font sizes.*/
		.ExternalClass {width:100%;}
		/* Force Hotmail to display emails at full width */
		.ExternalClass, .ExternalClass p, .ExternalClass span, .ExternalClass font, .ExternalClass td, .ExternalClass div {line-height: 100%;}
		/* Forces Hotmail to display normal line spacing.*/
		table td {border-collapse: collapse;}
		/* Outlook 07, 10 Padding issue fix */
		table { border-collapse:collapse; mso-table-lspace:0pt; mso-table-rspace:0pt; }
		/* Remove spacing around Outlook 07, 10 tables*/
		hr.style-six {border: 0;height: 0;border-top: 1px solid #dfdfdf;border-bottom: 1px solid rgba(255, 255, 255, 0.3);}
		@media (max-width:600px){
		* {-webkit-text-size-adjust: none !important;}
		.email_320_resize[class~=email_320_resize] {width:320px !important;}
		.width244 {width: 244px !important;}
		.displayblock_fullwidth[class~=displayblock_fullwidth] {display:block !important;width:100% !important;}
		.displayblock[class~=displayblock] {display:block !important;}
		.fullwidth[class~=fullwidth] {width:100% !important;}
		.mobilehide[class~=mobilehide]{display:none !important;;visibility:hidden !important;;}
		.mobileshow[class~=mobileshow]{display: block !important;max-height: inherit !important;overflow: visible !important;}
		.mobileheader[class~=mobileheader]{width:280px !important;height:185px !important;max-height:185px !important;}
		.noborder[class~=noborder]{border:none !important;}
		.paddingreset[class~=paddingreset]{padding:0 !important;}
		.horizontalpaddingreset[class~=horizontalpaddingreset]{padding-left:0px !important;padding-right:0px !important;}
		.resizeimage320[class~=resizeimage320]{height:320px !important;width:320px !important;}
		.resizeheader[class~=resizeheader]{height:92px !important;width:320px !important;}
		.titelresize[class~=titelresize]{font-size:24px !important;line-height:28px !important;}
		.textalignleft[class~=textalignleft]{text-align:left !important;}
		.padding20pxhorizontal[class~=padding20pxhorizontal]{padding-left:20px !important;padding-right:20px !important;}
		.paddingtopbot10px[class~=paddingtopbot10px]{padding-top:10px!important;padding-bottom:10px!important;}
		.paddingtop10px[class~=paddingtop10px]{padding-top:10px!important;}
		.paddingtop20px[class~=paddingtop20px]{padding-top:20px!important;}
		.paddingtop13px[class~=paddingtop13px]{padding-top:13px!important;}
		.scalefooterimage[class=scalefooterimage]{width:181px !important;height:51px !important;}
		.scalestappenimage[class~=scalestappenimage]{height: 53px !important;width: 320px !important;}
		.scalebanner[class~=scalebanner]{height: auto !important;width: 320px !important;}
		}
		</style>
	</head>
	<body class="lato"> 
		<table style="border-collapse:collapse; width:100%;background-color:white;">
			<tr>
				<td style="padding:0;background-color:#f0f1f3;height:50px;" align="center">
					<table style="border-collapse:collapse; width:640px;background-color:#f0f1f3;" class="email_320_resize">
						<tr>
							<td style="padding:0;font:1px/1px Arial;color:#f0f1f2;background-color:#f0f1f2" align="center" class="mobilehide">
								Bedankt voor het doorgeven.
							</td>
						</tr>
						<tr>
							<td style="background-color:#f0f1f3;padding:0 20px;" valign="top" align="right" class="paddingtopbot10px">
								<!-- partnerid / klantnummer -->
								<table style="border-collapse:collapse;width:400px" class="fullwidth" align="left">
									<tr>
										<td style="background-color:#f0f1f3;padding:0 35px 0px 0px;border-right: 1px solid #888" valign="top" align="right" class="paddingtop10px textalignleft noborder">
											<span style="font:normal 14px/14px Arial; color:#888;" class="lato">Uw klantnummer: 00000000000</span>
										</td>
									</tr>
								</table>
								<!-- //partnerid / klantnummer -->
								<!-- login mijn essent -->
								<table style="border-collapse:collapse;width:160px;" class="fullwidth" align="right">
									<tr>
										<td style="padding:0 10px 0 0;" align="left" valign="middle" class="mobilehide">
											<a href="https://web.flowmailer.net/link?id=x" style="border:0;" title="inloggen Mijn Essent" target="_blank"><img src="http://img.mailrd.net/essent/basis/icon_inloggen.png" width="16" height="16" style="display: block;" /></a>
										</td>
										<td style="padding:0;" align="left" valign="middle">
											<span style="font:normal 14px/14px Arial; color:#888;" class="lato">
												Inloggen <a href="https://web.flowmailer.net/link?id=x" style="text-decoration:none;color:#0098b3;" title="inloggen Mijn Essent" target="_blank">Mijn Essent</a>
											</span>
										</td>
									</tr>
								</table>
							</td>
						</tr>
					</table>
				</td>
			</tr>
			<tr>
				<td style="padding:0;" valign="top" align="center">
					<table style="border-collapse:collapse; width:640px; background-color:#ffffff" class="email_320_resize">
						<tr>
							<td style="padding:0" valign="top">
								<!-- header -->
								<table style="border-collapse:collapse;width:100%">
									<tr>
										<td style="padding:25px 20px 0 20px" valign="top">
											<table style="width:100%;border-collapse:collapse;">
												<tr>
													<td style="padding:0" valign="top">
														<table style="border-collapse:collapse;width:135px;display:block;" align="left" class="fullwidth">
															<tr>
																<td style="padding:0;" valign="top" align="left">
																	<a href="https://web.flowmailer.net/link?id=x" style="border:0;" title="Essent" target="_blank"><img src="http://img.mailrd.net/essent/basis/logo_essent.png" alt="Essent" width="135" height="77" border="0" style="display:block" /></a>
																</td>
															</tr>
														</table>
														<table style="border-collapse:collapse;width:460px;display:block;" align="left" class="fullwidth">
															<tr>
																<td style="padding:7px 0 0 0;background-color: white;width:1px" valign="top" align="left" class="mobilehide">
																	<img src="http://img.mailrd.net/essent/basis/trans.gif" height="70" width="1" style="display:block;"/>
																</td>
																<!---->
																<!--[if mso]><![endif]-->
															<!---->
																<td style="padding:10px 0 0 20px;" valign="middle" align="left" class="horizontalpaddingreset paddingtop13px lato">
																	<span style="font:normal 24px/30px Arial, sans-serif; color:#e60166;" class="titelresize">
																		We hebben uw meterstanden ontvangen
																	</span>
																</td>
															<!---->
															</tr>
														</table>
													</td>
												</tr>
											</table>
										</td>
									</tr>
								</table>
								<!-- //header -->	
								<!-- Body-->
								<table style="border-collapse:collapse; width:100%;">
									<tr>
										<td style="padding:20px 20px 0 20px" valign="top">
											<!-- Artikelen -->
											<table style="border-collapse:collapse;padding:0; width:100%;border-top: 1px solid #dfdfdf;">
												<!-- Artikel intro -->
												<tr>
													<td style="padding:25px 0 0 0;" valign="top" align="left" class="textalignleft">
														<table style="width:100%;border-collapse:collapse;">
															<tr>
																<td style="padding:0 0 20px 0;font:normal 16px/24px Arial; color:#777;" align="left" valign="top" class="lato">
																	Beste heer X,
																</td>
															</tr>
															
															<tr>
																<td style="padding:0 0 20px 0;font:normal 16px/24px Arial; color:#777;" align="left" valign="top" class="lato">
																	Wij hebben uw meterstand ontvangen.
																</td>
															</tr>
															<tr>
																<td style="padding:0 0 8px 0;font:normal 18px/24px Arial; color:#e60167;" align="left" valign="top" class="lato">
																	Uw gegevens
																</td>
															</tr>
															<tr>
																<td style="padding:0 0 20px 0;font:normal 16px/24px Arial; color:#777;" align="left" valign="top" class="lato">
																	Leveringsadres: Dam 1, 1001 AA AMSTERDAM<br/>
																	Opnamekenmerk: 000000000
																</td>
															</tr>
															
															
															<tr>
																<td style="padding:0 0 8px 0;font:normal 18px/24px Arial; color:#e60167;" align="left" valign="top" class="lato">
																	Uw meterstand
																</td>
															</tr>
															<tr>
																<td style="padding:0 0 20px 0;font:normal 16px/24px Arial; color:#777;" align="left" valign="top" class="lato">
																	<table style="padding:0;font:normal 16px/24px Arial;color:#777" class="lato">
																		<tr>
																			<td style="padding:0;width:250px;" align="left" valign="top">
																				<strong>Meter</strong>
																			</td>
																			<td style="padding:0;width:150px;" align="left" valign="top">
																				<strong>Meterstand</strong>
																			</td>												
																			<td style="padding:0;width:150px;" align="left" valign="top">
																				<strong>Opnamedatum</strong>
																			</td>													
																		</tr>				
							<tr>
																					<td style="padding:0;" align="left" valign="top">
																						Gas: 
																					</td>
																					<td style="padding:0;" align="left" valign="top">
																						3.14 m<sup>3</sup>
																					</td>															
																					<td style="padding:0 20px 0 0;" align="left" valign="top">
																						1 april 2020
																					</td>
																				</tr>
																	</table>
																</td>
															</tr>
														</table>
													</td>
												</tr>
												<!-- //Artikel intro -->	
												<!-- Afsluiter -->
												<tr>
													<td style="padding:30px 0 0 0;" valign="top" align="left" class="textalignleft">
														<table style="border-top: 1px solid #dfdfdf;width:100%;border-collapse:collapse;">
															<tr>
																<td style="padding:25px 0 0 0;font:normal 16px/24px Arial; color:#777;" align="left" valign="top" class="lato">
																	Met vriendelijke groet, 
																	<br/><br/>
																	Klantenservice Essent
																</td>
															</tr>
														</table>
													</td>
												</tr>
												<!-- //Afsluiter -->
												
												<!-- Banner -->
												<tr>
													<td style="padding: 0;">
														<!-- banner -->
														<table style="border-collapse: collapse; width: 100%;">
															<tr>
																<td style="padding:0 0 0px 0;font:normal 16px/24px Arial;color:#777" align="left" valign="top" class="lato">
																	<table style="border-collapse: collapse; width: 100%;">
																		<tr>
																			<td style="padding: 20px 0 0 0;" align="center">
<a href="https://web.flowmailer.net/link?id=x" title="thuisvoordeel banner" target="_blank"> <img src="http://img.mailrd.net/essent/thuisvoordeel_banner.jpg" width="600" style="display:block;height:auto" class="scalebanner" /> </a>																			</td>
																		</tr>
																	</table>
																</td>
															</tr>
														</table>	
														<!-- / banner -->
													</td>
												</tr>
												<!-- //Banner -->
												
												<tr>
													<td style="padding:0 0 25px 0" valign="top" align="left" class="textalignleft"><img src="http://img.mailrd.net/essent/basis/trans.gif" style="display: block;" height="1"/></td>
												</tr>
											</table>
											<!-- // Artikelen -->
										</td>
									</tr> 
								</table>
								<!-- Body-->
							</td>
						</tr>
					</table>
				</td>
			</tr>
			<!--FEEDBACK-->
			<tr>
				<td style="padding:0;" valign="top" align="center">
					<table style="border-collapse:collapse; width:640px;background-color:white;" class="width375">
						<tr>
							<td style="padding: 0 0 20px 0;" align="center">
								<table style="border-collapse:collapse; width:100%; border-top: 1px solid #dfdfdf">
									<tr>
										<td style="padding: 20px 0;" class="padding20pxhorizontal">
											<table style="border-collapse: collapse; width: 100%;" align="left">
												<tr>
													<td style="padding: 8px 0 0 0;font:normal 18px/24px Arial; color:#e60167;" align="center">
														Wat vindt u van deze e-mail?
													</td>
												</tr>
											</table>
										</td>
									</tr>
									<tr>
										<td style="padding: 0 0 20px 0;">
											<table style="border-collapse: collapse; width: 100%;" align="left">
												<tr>
													<td style="padding: 0;" class="paddingtop10 paddingleft30" align="center">
														<table style="border-collapse: collapse;">
															<tr>
																<td style="padding: 0 30px 0 0;" class="resetpadding">
<a href="https://web.flowmailer.net/link?id=x" style="border: 0;" title="Positief" target="_blank"> <img src="http://img.mailrd.net/essent/essent_feedback_positief.png" width="50" height="50" style="display: block; border:none;" alt="Positief" title="Positief" /> </a>																</td>
																<td style="padding: 0;">
<a href="https://web.flowmailer.net/link?id=x" style="border: 0;" title="Negatief" target="_blank"> <img src="http://img.mailrd.net/essent/essent_feedback_negatief.png" width="50" height="50" style="display: block; border:none;" alt="Negatief" title="Negatief" /> </a>																</td>
															</tr>
														</table>
													</td>
												</tr>
											</table>
										</td>
									</tr>
								</table>
							</td>
						</tr>
					</table>
				</td>
			</tr>
			<!--/FEEDBACK-->
			<!-- Serviceblok -->
			<tr>
				<td style="padding:0;background-color:#f0f1f3;" align="center">
					<table style="border-collapse:collapse; width:640px;background-color:#f3f1f2;" class="email_320_resize">
						<tr>
							<td style="padding:0 20px;" align="left" valign="top">
								<table style="width:100%;border-collapse:collapse;" align="left">
									<tr>
										<td style="padding:25px 0 8px 0;font:normal 18px/24px Arial; color:#e60167;" align="left" valign="top" class="lato">
											Heeft u een vraag?
										</td>
									</tr>
									<tr>
										<td style="padding:0;background-color:#f3f1f2;font:normal 16px/24px Arial; color:#777;" align="left" valign="top" class="lato">
											Wij helpen u graag. Bekijk alle
											<a href="https://web.flowmailer.net/link?id=x" style="color:#0098b3" title="veelgestelde vragen" target="_blank">veelgestelde vragen</a> 
											en vind snel en eenvoudig het antwoord op uw vraag. 
										</td>
									</tr>
								</table>
							</td>
						</tr>
					</table>
				</td>
			</tr>
			<!-- //Serviceblok -->
			<!-- Payoff -->
			<tr>
				<td style="padding:25px 0;background-color:#f0f1f3;" align="center">
					<table style="border-collapse:collapse; width:640px;background-color:#f3f1f2;" class="email_320_resize">
						<tr>
							<td style="padding:0 20px 0 20px;background-color:#f3f1f2;" valign="top" align="right">
								<img src="http://img.mailrd.net/essent/basis/payoff_logo_essent.png" width="264" height="80" style="display: block;"/>
							</td>
						</tr>
					</table>
				</td>
			</tr>
			<!-- //Payoff -->
			<!-- Disclaimer -->
			<tr>
				<td style="padding: 0; background-color: white;" align="center">
					<table style="border-collapse: collapse; width: 640px; background-color: white;" class="email_320_resize">
						<tr>
							<td style="padding: 30px 20px 30px 20px" valign="top" align="left" class="padding20pxhorizontal">
								<span style="font: bold 14px/18px Arial, sans-serif; color: #737373;">
									Copyright © 2020 Essent
								</span>
								<br/><br/>
								<span style="font: normal 14px/18px Arial, sans-serif; color: #acacac;">
									U ontvangt deze e-mail als klant van Essent. Via Mijn Essent kunt u uw gegevens en e&#8209;mailvoorkeuren wijzigen.
									Ontvangt u liever geen banners in onze e-mails? U kunt zich hiervoor <a href="https://web.flowmailer.net/link?id=x" style="color:#0098b3;text-decoration: none;" title="afmelden" target="_blank">afmelden</a>.
									<br/><br/>
									Persoonlijke gegevens van klanten worden met de grootst mogelijke zorgvuldigheid behandeld en beveiligd. 
									Essent houdt zich in alle gevallen aan de eisen die de Algemene Verordening Gegevensbescherming stelt. 
									Meer informatie hierover vindt u in onze <a href="https://web.flowmailer.net/link?id=x" style="color:#0098b3;text-decoration: none;" title="privacyverklaring" target="_blank">Privacyverklaring</a>.
									<br/><br/>
									Deze e-mail is een uitgave van Essent Retail Energie B.V. Postbus 1484, 5200 BM 's&#8209;Hertogenbosch.
									<p align="right">Referentienummer: F803</p>
								</span>
							</td>
						</tr>
					</table>
				</td>
			</tr>
			<!-- //Disclaimer -->
		</table>
<img src="https://web.flowmailer.net/open.gif?id=x" width="1" height="1" class="ot" />	</body>
</html>
