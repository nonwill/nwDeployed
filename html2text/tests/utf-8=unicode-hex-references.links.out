


                                       Bedankt voor het doorgeven.
Uw klantnummer: 00000000000

                                               _[_1_][2]  Inloggen _M_i_j_n_ _E_s_s_e_n_t[3]
                            _[_E_s_s_e_n_t_][4]
                            [5]  We hebben uw meterstanden ontvangen

                            Beste heer X,
                            Wij hebben uw meterstand ontvangen.
                            Uw gegevens
                            Leveringsadres: Dam 1, 1001 AA AMSTERDAM
                            Opnamekenmerk: 000000000
                            Uw meterstand
                            MMeetteerr MMeetteerrssttaanndd OOppnnaammeeddaattuumm
                            Gas:  3.14 m3    1 april 2020
                            Met vriendelijke groet,

                            Klantenservice Essent
                            _[_6_][7]

                            [8]
                                      Wat vindt u van deze e-mail?
                                      _[_P_o_s_i_t_i_e_f_][9] _[_N_e_g_a_t_i_e_f_][10]
      Heeft u een vraag?
      Wij helpen u graag. Bekijk alle _v_e_e_l_g_e_s_t_e_l_d_e_ _v_r_a_g_e_n[11] en vind snel en
      eenvoudig het antwoord op uw vraag.
                                                        [12]
 Copyright © 2020 Essent

 U ontvangt deze e-mail als klant van Essent. Via Mijn Essent kunt u uw
gegevens en e‑mailvoorkeuren wijzigen. Ontvangt u liever geen banners in onze
e-mails? U kunt zich hiervoor _a_f_m_e_l_d_e_n[13].

Persoonlijke gegevens van klanten worden met de grootst mogelijke
zorgvuldigheid behandeld en beveiligd. Essent houdt zich in alle gevallen aan
de eisen die de Algemene Verordening Gegevensbescherming stelt. Meer informatie
hierover vindt u in onze _P_r_i_v_a_c_y_v_e_r_k_l_a_r_i_n_g[14].

Deze e-mail is een uitgave van Essent Retail Energie B.V. Postbus 1484, 5200 BM
's‑Hertogenbosch.
                                                         Referentienummer: F803
[15]


############ RReeffeerreenncceess ############
   1. http://img.mailrd.net/essent/basis/icon_inloggen.png
   2. https://web.flowmailer.net/link?id=x
   3. https://web.flowmailer.net/link?id=x
   4. https://web.flowmailer.net/link?id=x
   5. http://img.mailrd.net/essent/basis/trans.gif
   6. http://img.mailrd.net/essent/thuisvoordeel_banner.jpg
   7. https://web.flowmailer.net/link?id=x
   8. http://img.mailrd.net/essent/basis/trans.gif
   9. https://web.flowmailer.net/link?id=x
  10. https://web.flowmailer.net/link?id=x
  11. https://web.flowmailer.net/link?id=x
  12. http://img.mailrd.net/essent/basis/payoff_logo_essent.png
  13. https://web.flowmailer.net/link?id=x
  14. https://web.flowmailer.net/link?id=x
  15. https://web.flowmailer.net/open.gif?id=x
